### ISSUE TITLE
Does this tool support   elasticsearch in aws eks fargate

### ISSUE THREAD

**Issue by Svastikkka** [aws/aws-for-fluent-bit#329]
### Describe the question/issue
Hi, I am trying to get the AWS EKS Fargate logs from the aws-logging config map to Elasticsearch  pod.

### Fluent Bit Configmap
`apiVersion: v1
data:
  filters.conf: |
    [FILTER]
      Name                kubernetes
      Match               kube.*
      Kube_URL            https://kubernetes.default.svc:443
      Kube_Tag_Prefix     kube.var.log.containers.
      Merge_Log           On
      Merge_Log_Key       log_processed
      K8S-Logging.Parser  On
      K8S-Logging.Exclude Off
  output.conf: |
    [OUTPUT]
      Name            es
      Match           *
      Host            http://es.observability.svc.cluster.local
      Port            9200
      Index           <index-name-{now/d}>
      Retry_Limit     20
      Replace_Dots    On
kind: ConfigMap
metadata:
  name: aws-logging
  namespace: aws-observability
`
Note:- Elasticsearch pod is in the observability namespace and config map is created in the aws-observability namespace.

**Comment by zhonghui12** [comment]
you could refer to this page for more config details: https://docs.fluentbit.io/manual/pipeline/outputs/elasticsearch

**Comment by matthewfala** [comment]
Closing due to inactivity. Please reopen this issue if you have any further comments or concerns.

**Comment by hustshawn** [comment]
> Yes and you could refer to this page for more config details: https://docs.aws.amazon.com/eks/latest/userguide/fargate-logging.html

Hi @zhonghui12 , from the document, it only shows the example for Amazon OpenSearch, instead of self-hosted ES. I tried as well, but seems cannot get logs output to ES. Since the Fargate builtin fluentbit, it's hard to debug what's going wrong. Have you tried it works?

@Svastikkka Do you get it working?

**Comment by Svastikkka** [comment]
Hi @hustshawn we decided to use a fluent bit as a sidecar
```
FROM fluent/fluent-bit:2.0
COPY fluent-bit.conf /fluent-bit/etc/fluent-bit.conf
COPY parsers.conf /fluent-bit/etc/parsers.conf
```

**Comment by hustshawn** [comment]
@Svastikkka thanks for letting me know. so do you collect logs from the local storage?

**Comment by Svastikkka** [comment]
Yes @hustshawn we store logs inside container and delete them after 2 days

**Comment by alexbumbacea** [comment]
It seems like fluentbit instance doesn't have access to svc.cluster.local stuff - not sure what is the error, as we can't retrieve the logs of that process. 
I managed to get a similar configuration working by exposing the ES via a NLB and using the NLB in the fluentbit config.

**Comment by Dashing-Nelson** [comment]
I am also trying to set it up using the above method. No luck

**Comment by obiomaokorowu** [comment]
Have anyone able to get this to work? I mean using Fluent-Bit and pushing the logs to opensearch either aws or locally hosted on eks??
